### PR TITLE
asf

### PR DESCRIPTION
**Pull request by kshitij430**

### CODE REVIEW

**Comment by kshitijwork1997** [comment]
Fix the error and create a PR. thanks

File: index.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
 const moment = require("moment");
-console.log("TODAYS TIME", moment());
+console.log("TODAYS TIME", asfsaf());
```
Comment (kshitijwork1997) [inline index.js:2]: method name is wrong, can you fix this?

**Comment by kshitij430** [comment]
> Fix the error and create a PR. thanks

**Review by kshitijwork1997 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (kshitij430) [reply index.js]: I have fixed it, can you check?

**Review by kshitijwork1997 — APPROVED** [review]
Thank you I approve this.

**Pull request merged**